test/config.yaml

```diff
@@ -0,0 +1,86 @@
+#### General Settings ####
+# location of system level settings
+settings:
+  title: My Very Cool Project
+  Author: Bob
+
+  # path like settings
+  data: /data/bob/original_data
```

Comment:
is this the path to fastqs? Or do indexes go here as well? I feel like this is too general of a setting, and that indexes, annotations, etc can just specify the full path. And maybe we can have a regexp for fastqs